### ISSUE TITLE
[a11y] /firefox/all/ page's disabled content missing `aria-disabled` attributes

### ISSUE THREAD

**Issue by reemhamz** [mozilla/bedrock#15065]
## Description

In the https://www.mozilla.org/en-US/firefox/all/ page, there are disabled elements that activate once a user goes through the selections. However, while the elements are disabled, they should hold the `aria-disabled` attribute with a value of `true`.  Once the element activates, the attribute should switch to have a value of `false`.

### Steps to reproduce

1. Go to https://www.mozilla.org/en-US/firefox/all/
2. Switch on screen reader device (I used VoiceOver on MacOS)
3. Tab over to disabled elements (don't select a browser in initial step)

### Expected result
Screen reader should tell the user that the element is dimmed (VoiceOver) or disabled (screen reader gets this information based on the `aria-disabled` attribute)

### Actual result
No mention of a dimmed or disabled feature on the disabled element

### Environment

Firefox v129, MacOS Sonoma
